src/core/ext/transport/binder/client/channel_create.cc

```diff
@@ -146,11 +146,13 @@ std::shared_ptr<grpc::Channel> CreateCustomBinderChannel(
     void*, jobject, absl::string_view, absl::string_view,
     std::shared_ptr<grpc::experimental::binder::SecurityPolicy>,
     const ChannelArguments&) {
+  gpr_log(GPR_ERROR, "This APK is compiled with Android API level = %d, which is not supported", __ANDROID_API__);
```

Comment:
maybe specify what is the minimum level here as well.

Reply:
Hard to get supported version here, let me refer the user to `port_platform.h` to check